udata/core/dataset/models.py

```diff
@@ -718,6 +718,11 @@ def quality(self):
 
         result['score'] = self.compute_quality_score(result)
         return result
+    
+    @property
+    def downloads(self):
+        return sum(map(lambda o: o.metrics.get('views', 0), self.resources))
```

Comment:
We mostly use list comprehension in udata code
```suggestion
        return sum(resource.metrics.get('views', 0) for resource in self.resources)
```